models/oauth2.go

```diff
@@ -125,8 +125,18 @@ func InitOAuth2() error {
 	if err := oauth2.Init(x); err != nil {
 		return err
 	}
-	loginSources, _ := GetActiveOAuth2ProviderLoginSources()
+	return initOuath2LoginSources()
+}
 
+// ResetOAuth2 clears existing OAuth2 providers and loads them from the DB
+func ResetOAuth2() error {
+	oauth2.ClearProviders()
+	return initOuath2LoginSources()
+}
+
+// initOuath2LoginSources is used to load and register all active OAuth2 providers
+func initOuath2LoginSources() error {
```

Comment:
typo, should be `initOAuth2LoginSources`

Reply:
Fixed